src/Components/CustomElements/src/js/WebpackBuild.targets

```diff
@@ -0,0 +1,38 @@
+<Project>
```

Comment:
We usually rely on shared infrastructure to avoid lots of disparate ways of doing things across the repo. We're not perfect in that regard but using a `*.npmproj` file remains what I suggest here.

Reply:
@dougbu we only use npmproj when the JS doesn't go into a nuget package, which is not the case here. Building the JS in the csproj is the most straightforward approach in this case, as we do for our auth packages. The npmproj only adds friction in this case.

FYI, we are going to be productionizing a package to better integrate these scenarios into RCLs, so we will likely get rid of all this custom MSBuild in favor of using a production ready version.